math/matrix4.cpp

```diff
@@ -129,6 +129,134 @@ void Matrix<4, 4>::invertAffineOrthonormal() {
 	setPosition(position);
 }
 
+/*
```

Comment:
from where it was taken? license?

Reply:
From a bugreport in Mesa. I suppose the license is public domain.

Anyway, it is not used at all. Removed.